engine/src/main/scala/cromwell/engine/workflow/workflowstore/SqlWorkflowStore.scala

```diff
@@ -68,13 +68,13 @@ case class SqlWorkflowStore(sqlDatabase: WorkflowStoreSqlDatabase) extends Workf
   private def toWorkflowStoreEntry(workflowSourceFiles: WorkflowSourceFilesCollection): WorkflowStoreEntry = {
     WorkflowStoreEntry(
       workflowExecutionUuid = WorkflowId.randomId().toString,
-      workflowDefinition = workflowSourceFiles.wdlSource.toClob,
-      workflowInputs = workflowSourceFiles.inputsJson.toClob,
-      workflowOptions = workflowSourceFiles.workflowOptionsJson.toClob,
-      customLabels = workflowSourceFiles.labelsJson.toClob,
+      workflowDefinition = workflowSourceFiles.wdlSource.toClobOption,
+      workflowInputs = workflowSourceFiles.inputsJson.toClobOption,
+      workflowOptions = workflowSourceFiles.workflowOptionsJson.toClobOption,
+      customLabels = workflowSourceFiles.labelsJson.toClob("{}"),
```

Comment:
This was a little confusing on skim-reading. Could you make it `...toClob(default = "{}")` for the benefit of Future Chris?